### PR TITLE
Add AdjustmentFilter

### PR DESCRIPTION
**Pull request by finscn**
This filter could help use to Adjust some basic properties of color : 
gamma, contrast, saturation, brightness, alpha & RGB Channel.

Some effects also could be done with `ColorMatrixFilter` , 
But `AdjustmentFilter` is simple & easy to be understood .

And It's more fast than `ColorMatrixFilter`. The core of the shader : 
```
    vec3 rgb = pow(c.rgb, vec3(1. / gamma));
    rgb = mix(vec3(.5), mix(vec3(dot(vec3(.2125, .7154, .0721), rgb)), rgb, saturation), contrast);
    rgb.r *= red;
    rgb.g *= green;
    rgb.b *= blue;
    c.rgb = rgb * brightness;
```
There is no any complicated matrix computation .

Online example: http://fatidol.com/pixi-filters/

### CODE REVIEW

File: filters/adjustment/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2013-2017 Mathew Groves, Chad Engler
```
Comment (bigtimebuddy) [inline filters/adjustment/LICENSE]: Change copyright?

File: filters/adjustment/src/AdjustmentFilter.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ * @param {number} [options.vignettingAlpha=1.0] - Amount of opacity of vignette
+ * @param {number} [options.vignettingBlur=0.3] - Blur intensity of the vignette
+ * @param {number} [seed=0] - A see value to apply to the random noise generation
+ */
+export default class AdjustmentFilter extends PIXI.Filter {
+    constructor(options) {
+        super(vertex, fragment);
+
+        Object.assign(this, {
+            gamma: 1,
```
Comment (bigtimebuddy) [inline filters/adjustment/src/AdjustmentFilter.js]: Need to document each of the properties, not just the constructor options.

File: filters/adjustment/LICENSE (hunk 1 above)
Comment (finscn) [reply filters/adjustment/LICENSE]: After all of my filters be merged , I will create a new PR for change all of them.

Comment (bigtimebuddy) [reply filters/adjustment/LICENSE]: Okay 👍

File: filters/adjustment/src/AdjustmentFilter.js (hunk 2 above)
Comment (finscn) [reply filters/adjustment/src/AdjustmentFilter.js]: I don't understand.... How to add documentation in Object.assign() ?

Comment (bigtimebuddy) [reply filters/adjustment/src/AdjustmentFilter.js]: I'm not sure if you need to add the name after `@member` or not if it's not attached to `this.gamma`. Test the documentation with `npm run docs` to preview.

```js
Object.assign(this, {
  /**
   * The amount of Gamma
   * @member {number} gamma
   * @default 1
   */
  gamma: 1,
```

Comment (finscn) [reply filters/adjustment/src/AdjustmentFilter.js]: can't run `npm run docs` I don't know why...

**Comment by bigtimebuddy** [comment]
Another awesome and useful filter. Thanks @finscn!

**Pull request merged**